chapter2/sec17-closed-sets.tex

```diff
@@ -12,6 +12,27 @@ \subsubsection{Problem}
 \subsubsection{Solution}
 From Theorem 17.2 we get that there exists a closed subset $C \subseteq X$ such that $A = C \cap Y$. Since $Y$ is closed in $X$, from Theorem 17.1(2), we get that $A = (C \cap Y) \subseteq X$ is also a closed subset of $X$.
 
+%%%%%%%%%%%%%%%%%%%%%%%%% 3 %%%%%%%%%%%%%%%%%%%%%%%%%
+
```

Comment:
For consistency, how about having "\setcounter{subsection}{2} % 3" etc everywhere, so that each problem is modular, independent of the other problems in the same file.

Reply:
Sure thing. I thought the eventual aim is to get rid of counters once all the solutions are in.